alpine/APKBUILD.in

```diff
@@ -28,20 +28,18 @@ source="$pkgname-$pkgver.tar.gz"
 builddir="$srcdir"/$pkgname-$pkgver
 
 _sbindir=/usr/lib/frr
-_sysconfdir=/etc/frr
```

Comment:
lots of developers probably have custom scripts to set this up for themselves.  Do we need to have them change anything?

Reply:
No, the configure script detects and removes the `/frr` (before re-adding it)

https://github.com/FRRouting/frr/pull/15243/files#diff-49473dca262eeab3b4a43002adb08b4db31020d190caaad1594b47f1d5daa810R47